code/game/objects/items/weapons/twohanded.dm

```diff
@@ -377,6 +377,37 @@
 		desc = "A makeshift spear with [C42] attached to it. Alt+click on the spear to set your war cry!"
 	update_icon()
 
+
+//GREY TIDE
+
+/obj/item/weapon/twohanded/spear/grey_tide
+	icon_state = "spearglass0"
+	name = "\improper Grey Tide"
+	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities among Nanotrasen military forces. Alt+click to bind it to yourself."
+	force_unwielded = 15
+	force_wielded = 25
+	throwforce = 20
+	throw_speed = 4
+	attack_verb = list("gored")
+	var/mob/living/spear_owner

```

Comment:
make this a list, and have the first if statement in afterattack check the user against the list. this would be so that you are not generating multiple factions to one mob if the spear trades hands between a few people and gets some use in.